CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins
 ### Fixed
 - Updated asset build automation for Alpine target to ensure curl and libcurl based metrics work.
 - `check-http.rb`: An empty response body when using `-w` no longer creates a potentially confusing `no implicit conversion of nil into String` error
+- `metrics-curl.rb`: Simpler curl dectection for `/bin/dash` support (default on Ubuntu LTS) (@elfranne)
```

Comment:
This should go under `## [Unreleased]` as we don't modify existing releases, I will update it for you.